package.json

```diff
@@ -14,6 +14,7 @@
     "precommit": "lint-staged --allow-empty && npm run prettier:write --workspaces --if-present",
     "prepush": "npm run test:eslint --workspaces --if-present",
     "build": "npm run build:packages && npm run build:example",
+    "buildCi": "CI=false && npm run build",
```

Comment:
Why buildCi is making CI=false ?

Reply:
Warnings during build step breaks application. 

We got error like this:

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/29872339/234804769-b2395253-1e58-4c00-a549-50ca8e76e578.png">

and also check this link

https://github.com/orgs/community/discussions/25228#discussioncomment-3246960